library/unwind/src/lib.rs

```diff
@@ -108,3 +108,27 @@ extern "C" {}
 #[cfg(all(target_os = "windows", target_env = "gnu", target_abi = "llvm"))]
 #[link(name = "unwind", kind = "static", modifiers = "-bundle")]
 extern "C" {}
+
+#[cfg(any(target_os = "freebsd", target_os = "netbsd"))]
+#[link(name = "gcc_s")]
```

Comment:
Several of the existing `#[link]` attributes use `kind = "static", modifiers = "-bundle"`, but to keep behavior unchanged I didn't do this in the new `#[link]` attributes.

Reply:
Those new targets don't need to use `kind = "static"`, as the dynamic library are part of the base system.

Reply:
On sparc64-unknown-openbsd libgcc is used, which is a static library afaik.